packages/manager/modules/pci/src/components/project/quota-region-header/header.html

```diff
@@ -1,12 +1,21 @@
-<oui-header-tabs class="mb-2">
-    <oui-header-tabs-item
-        text="{{:: 'pci_project_quota_header' | translate}}"
-        state="pci.projects.project.quota"
-        state-params="{ projectId: $ctrl.projectId }">
-    </oui-header-tabs-item>
-    <oui-header-tabs-item
-        text="{{:: 'pci_project_region_header' | translate}}"
-        state="pci.projects.project.regions"
-        state-params="{ projectId: $ctrl.projectId }">
-    </oui-header-tabs-item>
-</oui-header-tabs>
+<oui-page-header class="mb-3">
+    <oui-header-tabs >
+        <oui-header-tabs-item
+            text="{{:: 'pci_project_quota_header' | translate}}"
+            state="pci.projects.project.quota"
+            state-params="{ projectId: $ctrl.projectId }">
+        </oui-header-tabs-item>
+        <oui-header-tabs-item
+            text="{{:: 'pci_project_region_header' | translate}}"
+            state="pci.projects.project.regions"
+            state-params="{ projectId: $ctrl.projectId }">
+        </oui-header-tabs-item>
+    </oui-header-tabs>
+    <oui-guide-menu data-text="{{:: 'pci_project_guides_header' | translate }}">
+        <oui-guide-menu-item
+            href="{{ $ctrl.guideUrl }}"
```

Comment:
```suggestion
            data-href="{{ $ctrl.guideUrl }}"
```

Because `href` does not refer to the actual `href` property. If you had `id` then you wouldn't have to add the `data-` 😅